react/legacy/index.js

```diff
@@ -0,0 +1,140 @@
+import React, { Fragment, useEffect, useState } from 'react'
```

Comment:
This file was mostly just moved from `react` to `react/legacy`, please refrain from delving too deeply into it for this PR